AutomatedTesting/Gem/PythonTests/terrain/EditorScripts/TerrainPhysicsCollider_ChangesSizeWithAxisAlignedBoxShapeChanges.py

```diff
@@ -0,0 +1,86 @@
+"""
+Copyright (c) Contributors to the Open 3D Engine Project.
+For complete copyright and license terms please see the LICENSE at the root of this distribution.
+
+SPDX-License-Identifier: Apache-2.0 OR MIT
+"""
+
+#fmt: off
+class Tests():
+    create_test_entity                      = ("Entity created successfully",    "Failed to create Entity")
+    add_physx_heightfield_collider          = ("PhysX Heightfield Collider component added", "Failed to add PhysX Heightfield Collider component")
```

Comment:
This line shouldn't be here, since we're not adding the PhysX Heightfield Collider any more.

Reply:
Done.